### DISCUSSION TITLE
Get directory list and show them in Select field

### DISCUSSION THREAD

**Discussion by vmind** [laravel/nova-issues#5596] [Q&A] [answered]
Hi,
can anyone help me with this.
How can I get directory list from public path and them list them using Select in resource.
 Also then I need get list of all files from selected dir and check theirs names.

Thanks

**Comment by crynobone** [comment, marked as answer]
Hi there, I believe `Illuminate\Filesystem\Filesystem` would offer such feature: https://laravel.com/api/10.x/Illuminate/Filesystem/Filesystem.html
